### DISCUSSION TITLE
Is optimizePackageImports going to be available for pages router?

### DISCUSSION THREAD

**Discussion by maverickdude** [vercel/next.js#55733] [Help]
### Summary

Hi everyone!

As I understand the new feature [optimizePackageImports](https://nextjs.org/docs/app/api-reference/next-config-js/optimizePackageImports) is only for the "app" router.

Is the same going to be available for the "pages" router some time soon?
Or is there any alternative way already available to implement in the "pages" router?

I want it mainly for 2 packages: `lodash` and `semantic-ui-react`.

Thank you.

**Comment by CinArb2** [comment]
I think this option is available for both, the app and the pages router [https://github.com/vercel/next.js/blob/canary/test/development/basic/barrel-optimization.test.ts#L76](https://github.com/vercel/next.js/blob/canary/test/development/basic/barrel-optimization.test.ts#L76)
